4. Angular esercizio 2/esercizio-due/src/app/app.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component } from '@angular/core';
+import { IObj, Gender, Role } from '../../../../1. Typescript esercizio 1/start';
+
+@Component({
+  selector: 'es2-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css']
+})
+export class AppComponent {
+  title = 'esercizio-due';
+  users: IObj[] = [
```

Comment:
IObj è veramente poco significativo come nome, inoltre l'esercizio non diceva di inserire l'oggetto (non un array) in app-component